src/types/widget.ts

```diff
@@ -44,13 +47,10 @@ export type UiState = {
   hierarchicalMenu?: {
     [attribute: string]: string[];
   };
-  refinementList?: {
-    [attribute: string]: string[];
-  };
-  numericRefinementList?: {
+  numericMenu?: {
     [attribute: string]: number;
```

Comment:
If I'm not mistaken the `numericMenu` takes a `string` not a `number`.

Reply:
Ah, you're right.